### ISSUE TITLE
ReactCasts #12 - `NODE_ENV` is "undefined"

### ISSUE THREAD

**Issue by metasean** [cassiozen/ReactCasts#39]
I initially hand-coded along with the video, but ended up with errors that I couldn't fix.  

Assuming I still had a typo lurking somewhere, I cloned, installed, and tried to run your code directly, only to get the same errors.

**Steps to reproduce:**

1. `git clone git@github.com:cassiozen/ReactCasts.git`
2. `cd ReactCasts/episode12`
3. `npm install`
4. `npm run start-dev`

**_Initial_ Error that I'm getting if I do the above steps in Atom's platform-ide-terminal:**
(Not sure why it makes a difference, but since it does, maybe it's an important clue.  And, technically, I did steps 1 & 2 in Terminal, then did an `atom .`, then did steps 3 & 4 in Atom.)
```
    ERROR in ./src/browser/index.js
    Module build failed: Error: Using `babel-preset-react-app` requires that yo
u specify `NODE_ENV` or `BABEL_ENV` environment variables. Valid values are "de
velopment", "test", and "production". Instead, received: "undefined". (While pr
ocessing preset:/Users/metasean/dev/ReactCasts/ReactCasts/episode12/node_module
s/babel-preset-react-app/index.js")
```

**_Subsequent_ Error that I'm getting if I do the above steps in Atom's platform-ide-terminal:**
**_AND_**
**Error that I get if I do all of the above steps directly in a normal OS X terminal**
```
/Users/metasean/dev/ReactCasts/ReactCasts/episode12/server.js:71
throw new Error("Module build failed: Error: Using `babel-preset-react-app` req
uires that you specify `NODE_ENV` or `BABEL_ENV` environment variables. Valid v
alues are \"development\",\"test\", and \"production\". Instead, received: \"un
defined\". (While processing preset:\"/Users/metasean/dev/ReactCasts/ReactCasts
/episode12/node_modules/babel-preset-react-app/index.js\")
```

FWIW: my `node --version` is `v8.1.3`

**Comment by cassiozen** [comment]
Hi @metasean - Are you using the code from the repo? Can you share your start-dev script?

**Comment by cassiozen** [comment]
@metasean - Did you get this to work?

**Comment by metasean** [comment]
@cassiozen - I think there was a common dependency issue.

I had initially replicated the problem using your repo.  Just now I cloned a fresh copy, npm installed, and started it again; and this time it worked.  

So of course, I tried to start mine, but got the same error as before.  _But this time, after I ran npm install again, it started up as expected._

Thanks for following up!  👍  (I've been bouncy between projects and just hadn't made it back here. :-( )
